packages/push-notifications/src/messages/PushNotificationsGetDeviceInfoMessage.ts

```diff
@@ -0,0 +1,13 @@
+import { AgentMessage } from '@aries-framework/core/build/agent/AgentMessage'
+import { Equals } from 'class-validator'
+
+/**
+ * Message to get the device information from another agent for push notifications
+ *
+ * @todo ADD RFC
+ */
+export class PushNotificationsGetDeviceInfoMessage extends AgentMessage {
```

Comment:
This is missing the device info message: https://didcomm.org/push-notifications/1.0/device-info  as response to get-device-info. As that is a message you receive as client we need to add the message and also a handler for it